### DISCUSSION TITLE
Adapting unicorn to masonite web framework

### DISCUSSION THREAD

**Discussion by ystekno** [MasoniteFramework/masonite#641] [Ideas]
With **django-unicorn**, it is possible to create reactive components such as react and angular without using javascript, jquery. Could you contact **Adam Hill** and start a study to apply this amazing package to the **masonite web framework**?

**Comment by Marlysson** [comment]
Would masonite-inertia be a replacement for It?

**Reply by ystekno** [reply]
Thanks a lot for your answer. As far as I can see masonite-inertia needs nodejs environment and vue.js. Django-unicorn uses **native python code** directly **without using javascript code**.

**Reply by girardinsamuel** [reply]
Yes that's two different approaches. Exactly like Livewire and Laravel-Inertia for Laravel. For now there is indeed now Masonite equivalent of Livewire. It would be a huge task to create this. I like it and would like to do it, unfortunately, we already have so much work with Masonite core ecosystem that I cannot find time to build this. At least for now...

If you want to take a crack at it or if someone wants to do it, we would be glad to help with @josephmancuso.
